doc/api/tls.md

```diff
@@ -1094,6 +1094,37 @@ See
 [SSL_get_shared_sigalgs](https://www.openssl.org/docs/man1.1.1/man3/SSL_get_shared_sigalgs.html)
 for more information.
 
+### `tlsSocket.exportKeyingMaterial(length, label)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `length` {number}
+  number of bytes to retrieve from keying material
+* `label` {string} an application specific label, typically this will be a
+value from the IANA Exporter Label Registry
+(https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#exporter-labels).
+
+* Returns: {Buffer} {number} of bytes of the keying material
+
+Example
+
+```js
+const keyingMaterial = tlsSocket.exportKeyingMaterial(
+  128,
+  'ttls keying material');
```

Comment:
```suggestion
  'tls keying material');
```

Reply:
ttls is correct. see https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#exporter-labels

Reply:
A comment indicating that would be nice. As it is, this gives the appearance of a typo

Reply:
I've changed it to "client finished" to make it more clear :)